src/GafferUI/StandardStyle.cpp

```diff
@@ -1389,7 +1389,7 @@ static const std::string &fragmentSource()
 		"	if( textureType==1 )"
 		"	{"
 		"		OUTCOLOR = texture2D( texture, gl_TexCoord[0].xy );"
-		"		OUTCOLOR = vec4( ieLinToSRGB( OUTCOLOR.r ), ieLinToSRGB( OUTCOLOR.g ), ieLinToSRGB( OUTCOLOR.b ), ieLinToSRGB( OUTCOLOR.a ) );"
+		"		OUTCOLOR = vec4( OUTCOLOR.a * ieLinToSRGB( OUTCOLOR.r / OUTCOLOR.a ), OUTCOLOR.a * ieLinToSRGB( OUTCOLOR.g / OUTCOLOR.a ), OUTCOLOR.a * ieLinToSRGB( OUTCOLOR.b / OUTCOLOR.a ), OUTCOLOR.a );"
```

Comment:
Is this technically undefined behaviour because we're not guarding against division by zero?

https://community.khronos.org/t/divide-by-zero/63081/2

I can't find the link now, but I also read somewhere that NVIDIA and Intel drivers treat this differently...

Reply:
I went to fix this, but then realized that we do already reject samples with zero alpha on line 1383